### PR TITLE
Fix system apps detection

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
The PR fixes system applications detection on Simulator. Please do not merge it until https://github.com/appium/node-simctl/commit/a5c4bfb33fcaf5139cbfeccc62c3dc49d005f06c is published

### CODE REVIEW

**Comment by imurchie** [comment]
`node-simctl@3.11.0` has the fix.

File: lib/simulator-xcode-8.js [hunk 1]
```diff
@@ -94,15 +94,22 @@ class SimulatorXcode8 extends SimulatorXcode7 {
...
-      let appContainer = await getAppContainer(this.udid, bundleId, false);
+      const appContainer = await getAppContainer(this.udid, bundleId, false);
       return appContainer.endsWith('.app');
     } catch (err) {
-      return false;
+      // get_app_container subcommand fails for system applications,
+      // so we try the hidden appinfo subcommand, which prints correct info for
+      // system/hidden apps
+      try {
+        const appInfo = await appInfo(this.udid, bundleId);
```
Comment (imurchie) [inline lib/simulator-xcode-8.js]: Does this command _not_ work for regular apps? Why not use it for everything?

Comment (mykola-mokhnach) [reply lib/simulator-xcode-8.js]: it does work, but the fact that this command is hidden makes me a bit suspicious about it. The stuff might stop working completely Apple decides to change the behaviour there in some future releases. Although the current approach will still work for non-system apps, which is the most common clients case.

**Comment by imurchie** [comment]
Published in `appium-ios-simulator@2.2.5`.

**Pull request merged**